tools/util/ReplayInterpreter.cpp

```diff
@@ -618,7 +631,7 @@ void ReplayInterpreter::replay_makeAllocator( void )
     }
     else {
       std::cerr << "Unknown class (" << type << "), skipping.\n";
-      return;
+      exit(1);
```

Comment:
As above, these should `UMPIRE_ERROR` (for consistency)

Reply:
As stated above, I'm not sure I agree for this tool.

Reply:
See my other comment 😄

Reply:
Yes, once we decide what to do, I will make the change everywhere.  This is an application.  Is UMPIRE_ERROR available to users of Umpire?

Reply:
It is, but you don't need to use it. I just think having a macro for this "print to cerr, exit(1)" pattern is a good idea.

Reply:
Done, please mark resolved.